content/02.introduction.md

```diff
@@ -121,6 +121,16 @@ Pathogenesis is made up of three major components: entry, replication, and sprea
 #### Immunological Mechanisms of Coronavirus-driven Disease in Humans
 
 Coronaviruses are known to cause respiratory illnesses in humans through the following possible mechanisms...[Summarize relevant mechanisms for cell entry & address evidence for/against ACE2 being important, etc.]
+One way that humans respond to viral threats is through the immune response.
+The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
+The innate immune system consists of barriers such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
```

Comment:
```suggestion
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
```
I kept reading this as neutrophils and macrophages were also barriers -- I think the extra comma helps make it clear that the skin is the barrier you're talking about!